scan/src/utils/ExecutableParser.re

```diff
@@ -0,0 +1,49 @@
+let checker = (str: string) => {
+  let reg = ".*=[$][0-9]+" |> Js.Re.fromString;
+  let t =
+    reg
+    |> Js.Re.exec_(_, str)
+    |> Belt_Option.mapWithDefault(_, [||], Js.Re.captures)
+    |> Belt_Array.length;
+  t > 0;
+};
+
+let getElementInList = (l, idx) => {
+  Belt_List.get(l, idx) |> Belt_Option.getExn;
+};
+
+let splitToPair = s => {
+  let tmp = s |> String.split_on_char('=');
+  let s0 = getElementInList(tmp, 0);
+  let s1 =
+    getElementInList(tmp, 1)
+    |> String.split_on_char('$')
+    |> getElementInList(_, 1)
+    |> int_of_string;
```

Comment:
This will crash if number is too big
```
symbol=$123123091823901283019381
```

Reply:
`If the input exceeds max_int it is converted to the signed integer min_int + input - max_int - 1. `